apps/openmw/mwworld/projectilemanager.cpp

```diff
@@ -202,6 +203,12 @@ namespace MWWorld
 
         osg::ref_ptr<osg::Node> projectile = mResourceSystem->getSceneManager()->getInstance(model, attachTo);
 
+        osg::ref_ptr<osg::ComputeBoundsVisitor> boundVisitor = new osg::ComputeBoundsVisitor();
```

Comment:
FYI, visitors can be created on the stack, so that line can be shortened to:

osg::ComputeBoundsVisitor boundVisitor;

You can do that with many other OSG objects, as long as you never create a ref_ptr of them.